vba_timeseries/models/exp/m_setup.m

```diff
@@ -0,0 +1,57 @@
+function [vo] = m_setup(vo)
+% Setup model function
+% SUUVID 2PL model where time elapsed controls the 'ability' axis (difficulty) and value scales the slope (discrimination)
+
+if nargin < 1, vo = struct(); end
+
+%% Model overview
+% evolution function
+vo.evo_fname = @m_suuvid_evo_base; %shared function in root of models
+
+% observation function
+vo.obs_fname = @m_suuvid_obs_exp;
+
+% number of hidden states
+vo.hidden_states = 2; %two-action approach for now
+
+% names of hidden states
+vo.state_names = {'Q1', 'Q2'};
+
+% number of model outputs (size of y vector)
+vo.n_outputs = 3; %two actions + no response
+
+% names of model outputs
+vo.y_names = {'y1', 'y2', 'none'};
+
+% number of free parameters in the evolution function
+vo.n_theta = 1;
+
+% names of parameters in the evolution function
+vo.theta_names = {'alpha'};
+
+% number of parameters in the observation function
+vo.n_phi = 4;
+
+% names of parameters in the observation function
+vo.phi_names = {'gamma', 'nu', 'kappa', 'omega'};
+
+%% Model priors
+
+priors.muTheta = zeros(vo.n_theta, 1);
+priors.SigmaTheta = 1e1*eye(vo.n_theta); %variance of 10 on all
+
+priors.muPhi = [ 0; ... %gamma: exponentiates to 1
```

Comment:
Maybe update comments to be something like, "gamma: exponential transform = 1" and the same for nu since its zero prior is transformed to 1.0 internally with exp(0).